tree/ntuple/v7/inc/ROOT/RNTupleMetrics.hxx

```diff
@@ -155,6 +158,74 @@ public:
 };
 
 
+// clang-format off
+/**
+\class ROOT::Experimental::Detail::RNTupleCalcFloatPerf
+\ingroup NTuple
+\brief A metric element that computes its floating point value from other counters.
+*/
+// clang-format on
+class RNTupleCalcIntPerf : public RNTuplePerfCounter {
```

Comment:
It would be good to have a small unit test for this counter type in `ntuple_metrics.cxx`